webpack.common.js

```diff
@@ -21,7 +28,32 @@ module.exports = {
     new HtmlWebpackPlugin({
       title: "Show Admin",
       template: "./src/index.ejs"
-    })
+    }),
+    ...("SENTRY_AUTH_TOKEN" in process.env &&
```

Comment:
add sentry env variables to https://github.com/fntechgit/summit-admin/blob/master/.env.example